src/Symfony/Component/Finder/Finder.php

```diff
@@ -85,6 +85,18 @@ public function __construct()
     }
 
     /**
+     * Get lookup directories
+     *
+     * @return array List of directories
+     *
+     * @api
+     */
+    public static function getDirs()
```

Comment:
I didn't make it static. this is an IDE autocompletion typo.
To conclude. i can't access the dirs attribute and i can't extend the class to adapt it to my use case!